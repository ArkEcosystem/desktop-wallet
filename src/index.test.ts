jest.mock("react-dom", () => ({ render: jest.fn() }));

describe("Application root", () => {
	it("should render without crashing", () => {
		const ReactDOM = require("react-dom");
		const div = document.createElement("div");
		div.id = "root";
		document.body.appendChild(div);
		require("./index");

		expect(ReactDOM.render).toHaveBeenCalled();
		expect(ReactDOM.render).toMatchSnapshot();
	});
});
