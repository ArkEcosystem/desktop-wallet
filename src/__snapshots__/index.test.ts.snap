// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application root should render without crashing 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      <HashRouter>
        <withRouter() />
      </HashRouter>,
      <div
        id="root"
      />,
    ],
  ],
  "results": Array [
    Object {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
